Update docs for new features and UI (#313)
statelyai/docs: docs/generate-test-paths.mdx

title: Generate test paths
---

> You can generate test path code for your state machines from the <ClipboardCheck size={18}/> **Tests** panel, which you can use to implement tests in your code. Both setup code and test code for each path are generated.

Reviewer: ```suggestion
You can generate test path code for your state machines from the <ClipboardCheck size={18}/> **Tests** panel, which you can use to implement tests in your code. Both the setup code and test code for each path are generated.
```

> This feature is in beta and [we’d love your feedback](https://discord.stately.ai).

Reviewer: ```suggestion
This feature is in beta and [we’d love your feedback](https://feedback.stately.ai).
```

You can choose between two path generation strategies:

- **Shortest path**: Generate paths that cover the shortest possible paths through your state machine and minimize the number of actions required to test functionality.

> - **Simplest path**: Generate paths that cover the simple paths, without repeated states, through your state machine and help verify that basic functionality is working as expected.

Reviewer: Clarifying as states can repeat with the current algorithm (good thing)

```suggestion
- **Simple path**: Generate paths that cover the simple paths, without repeated transitions, through your state machine and help verify that basic functionality is working as expected.
```
